packages/aws_signature_v4/pubspec.yaml

```diff
@@ -19,9 +19,9 @@ dependencies:
   meta: ^1.7.0
   path: ^1.8.0
 
-dependency_overrides:
-  aws_common:
-    path: ../aws_common
+# dependency_overrides:
```

Comment:
Why are we leaving this commented out?

Reply:
Just so I don't have to type it out when I'm developing. And because I can't use `pubspec_overrides` in main yet.